spec/filters/cidr_spec.rb

```diff
@@ -35,6 +36,75 @@
        insist { subject["tags"] }.nil?
     end
   end
+ 
+  # Test multple CIDR blocks passed into 'network'.  Make sure we try an
+  # IP in every range.
+
+  describe "IPV4 match, passing a list to network [192.168.0.30]" do
+   config <<-CONFIG
+       filter {
+        cidr {
+          address => [ "%{clientip}"]
+          network => [ "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
+          add_tag => [ "matched" ]
+        }
+      }
+    CONFIG
+
+    sample("clientip" => "192.168.0.30") do
```

Comment:
could this test and the one after this merge into the same one? they use the same configuration.